src/App.tsx

```diff
@@ -1,18 +1,40 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+  const [visibleItems, setVisibleItems] = useState(items.slice(0, perPage));
```

Comment:
better create a variable and assign it to useState
```suggestion
  const [visibleItems, setVisibleItems] = useState(someVariable);
```